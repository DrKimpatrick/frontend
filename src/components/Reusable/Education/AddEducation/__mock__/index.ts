export const initialState = {
  users: {
    currentUser: {
      isLoggedIn: true,
      loading: false,
      error: null,
      message: null,
      created: false,
      data: {
        firstName: 'John',
        lastName: 'Smith',
        profilePicture:
          'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png'
      }
    },
    user: {
      _id: 'id',
      username: 'username',
      email: 'email',
      roles: ['talent'],
      updatedAt: '2020-10-10',
      dateRegistered: '2020-10-10',
      paymentStatus: 'verified',
      featureChoice: 'local',
      verified: true
    }
  },
  messages: {
    message: 'sent'
  },
  educations: {
    loading: false,
    errors: {
      errors: null
    },
    education: {
      _id: 'id',
      schoolName: 'school name',
      level: 'level',
      degreeOrCertification: 'education',
      specializations: ['specialization'],
      startDate: '2020-10-10',
      endDate: '2020-10-5',
      accomplishments: ['snow'],
      isCurrentEducation: false,
      verificationStatus: 'verified'
    }
  }
};
