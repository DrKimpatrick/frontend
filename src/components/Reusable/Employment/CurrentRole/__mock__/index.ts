export const initialState = {
  users: {
    currentUser: {
      isLoggedIn: true,
      loading: false,
      error: null,
      message: null,
      created: false,
      data: {
        firstName: 'John',
        lastName: 'Smith',
        profilePicture:
          'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png'
      }
    },
    user: {
      _id: 'id',
      username: 'username',
      email: 'email',
      roles: ['talent'],
      updatedAt: '2020-10-10',
      dateRegistered: '2020-10-10',
      paymentStatus: 'verified',
      featureChoice: 'local',
      verified: true
    }
  },
  employments: {
    employment: {
      companyName: 'snow'
    },
    errors: null,
    loading: false
  },
  messages: {
    message: 'sent'
  },
  educations: {
    education: {
      schoolName: 'schoolName'
    }
  }
};
