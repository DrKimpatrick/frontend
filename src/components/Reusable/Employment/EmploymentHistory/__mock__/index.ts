export const initialState = {
  users: {
    currentUser: {
      isLoggedIn: true,
      loading: false,
      error: null,
      message: null,
      created: false,
      data: {
        firstName: 'John',
        lastName: 'Smith',
        profilePicture:
          'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png'
      }
    },
    user: {
      _id: 'id',
      username: 'username',
      email: 'email',
      roles: ['talent'],
      updatedAt: '2020-10-10',
      dateRegistered: '2020-10-10',
      paymentStatus: 'verified',
      featureChoice: 'local',
      verified: true
    }
  },
  employments: {
    employment: {
      companyName: 'snow',
      supervisor: 'supervisor',
      title: 'title',
      startDate: '2020-10-5',
      responsibilities: ['res'],
      accomplishments: ['res'],
      favoriteProject: 'res',
      skillsUsed: []
    },
    errors: null,
    loading: false
  },
  messages: {
    message: 'sent'
  },
  educations: {
    education: {
      schoolName: 'schoolName'
    }
  },
  skills: {
    userSkill: [
      {
        supervisor: {
          detail: {
            name: 'user',
            email: 'email@gmail.com',
            phoneNumber: '0786601003'
          },
          name: 'HR'
        },
        skillsUsed: [],
        responsibilities: [],
        accomplishments: [],
        verificationStatus: 'unverified',
        isCurrentPosition: true,
        _id: '5fcf65a2c4c797a20acc1c99',
        title: 'Product Manager',
        startDate: '2020-10-15T00:00:00.000Z',
        endDate: null,
        companyName: 'heza',
        favoriteProject: 'tembea',
        userId: '5fcdf47ea1d1dd1564a21613',
        createdAt: '2020-12-08T11:38:10.729Z',
        updatedAt: '2020-12-08T11:38:10.729Z'
      }
    ]
  }
};
