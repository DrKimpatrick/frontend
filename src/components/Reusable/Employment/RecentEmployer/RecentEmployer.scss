.recent-employer-section {
  position: relative;
  .title {
    font-size: 22px;
    color: #747474;
    text-decoration: none solid rgb(116, 116, 116);
  }

  .back-arrow {
    position: absolute;
    color: #747474;
    left: -43px;
    top: 0px;
    font-size: 22px;
  }

  .input-height {
    height: 42px !important;
    border-radius: 4px;
    color: #747474;
    &::placeholder {
      color: #c9c9c9;
    }
  }

  .input-height-input {
    color: #747474;
    &::placeholder {
      color: #c9c9c9;
    }
  }
}

@media (max-width: 786px) {
  .recent-employer-section {
    padding: 10px !important;

    .containers {
      width: 60%;
    }

    .back-arrow {
      display: none;
    }

    .title {
      width: 100% !important;
      text-align: center;
    }
  }
}

@media (max-width: 500px) {
  .recent-employer-section {
    margin-bottom: 40px !important;
    .containers {
      width: 100%;
    }
  }
}
