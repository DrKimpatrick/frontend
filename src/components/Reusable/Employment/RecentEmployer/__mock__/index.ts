export const initialState = {
  users: {
    currentUser: {
      isLoggedIn: true,
      loading: false,
      error: null,
      message: null,
      created: false,
      data: {
        firstName: 'John',
        lastName: 'Smith',
        profilePicture:
          'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png'
      }
    },
    user: {
      _id: 'id',
      username: 'username',
      email: 'email',
      roles: ['talent'],
      updatedAt: '2020-10-10',
      dateRegistered: '2020-10-10',
      paymentStatus: 'verified',
      featureChoice: 'local',
      verified: true
    }
  },
  employments: {
    employment: {
      companyName: 'snow',
      supervisor: 'supervisor',
      title: 'title',
      startDate: '2020-10-5',
      responsibilities: ['res'],
      accomplishments: ['res'],
      favoriteProject: 'res',
      _id: 'id'
    },
    errors: null,
    loading: false,
    submitLoading: false
  },
  messages: {
    message: 'sent'
  },
  educations: {
    education: {
      schoolName: 'schoolName'
    }
  },
  skills: {
    userSkill: [
      {
        skill: {
          _id: '5fcf1c5b11c16c41f7c6c723',
          skill: 'Python',
          createdAt: '2020-12-08T06:25:31.128Z',
          updatedAt: '2020-12-08T06:25:31.128Z'
        },
        level: 'intermediate',
        verificationStatus: 'verified',
        _id: '5fcf52105cd56b9843252696',
        createdAt: '2020-12-08T10:14:40.231Z',
        updatedAt: '2020-12-08T13:05:46.873Z'
      }
    ]
  }
};
