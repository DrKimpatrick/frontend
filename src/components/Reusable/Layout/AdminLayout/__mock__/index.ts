export const initialState = {
  users: {
    currentUser: {
      isLoggedIn: true,
      loading: false,
      error: null,
      message: null,
      created: false,
      data: {
        firstName: 'John',
        lastName: 'Smith',
        profilePicture:
          'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png'
      }
    },
    user: {}
  }
};
