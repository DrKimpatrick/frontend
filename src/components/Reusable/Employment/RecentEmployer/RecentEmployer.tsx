import React, { FC, useEffect } from 'react';
import { withRouter, useHistory } from 'react-router-dom';
import { useSelector, useDispatch } from 'react-redux';
import ArrowBackTwoToneIcon from '@material-ui/icons/ArrowBackTwoTone';
import NavBar from 'components/Reusable/Layout/NavBar/NavBar';
import { addEmployment } from 'redux/actions/employment';
import { EmploymentInput, InitialEmploymentValue } from 'components/Reusable';
import { RootState } from 'redux/store';
import { MainBackground } from 'components/Reusable/Layout/MainBackground';
import './RecentEmployer.scss';

const RecentEmployer: FC = () => {
  const dispatch = useDispatch();

  const history = useHistory();

  const reducer = useSelector((state: RootState) => {
    const { loading, errors, employment } = state.employments;
    const { message } = state.messages;
    return { message, loading, errors, employment };
  });

  useEffect(() => {
    if (reducer.message && reducer.employment) {
      history.push(`/employment-history/${reducer.employment._id}`);
    }
  }, [reducer.message, reducer.employment, history]);

  return (
    <>
      <NavBar />
      <div className="containers">
        <div className="recent-employer-section w-1/3 m-auto text-textGray">
          <div
            className="flex relative h-auto my-8"
            style={{ flexDirection: 'column' }}
          >
            <div style={{ display: 'flex' }}>
              <div
                className="back-arrow cursor-pointer"
                onClick={() => history.push('/skill-ranking')}
              >
                <ArrowBackTwoToneIcon />
              </div>
              <h1 className="font-bold text-base title">
                Who was your most recent employer?
              </h1>
            </div>

            <EmploymentInput
              initialValue={{
                companyName: '',
                title: '',
                supervisor: '',
                isCurrentPosition: false,
                responsibilities: [],
                skillsUsed: [],
                favoriteProject: '',
                accomplishments: [],
                responsibility: '',
                accomplishment: '',
                startDate: '',
                endDate: ''
              }}
              submit={(values: InitialEmploymentValue) =>
                addEmployment(values)(dispatch)
              }
              backendErrors={
                reducer.errors && reducer.errors.errors && reducer.errors.errors
              }
              loading={!!reducer.loading}
              buttonName="Next"
            />
          </div>
        </div>
      </div>
      <MainBackground />
    </>
  );
};

export default withRouter(RecentEmployer);
