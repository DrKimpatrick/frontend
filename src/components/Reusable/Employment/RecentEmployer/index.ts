import RecentEmployer from './RecentEmployer';

export { RecentEmployer };
