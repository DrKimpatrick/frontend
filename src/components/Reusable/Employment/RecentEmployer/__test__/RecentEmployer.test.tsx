import { cleanup, fireEvent, waitFor, render } from '@testing-library/react';
import React from 'react';
import ReactDom from 'react-dom';
import { BrowserRouter as Router } from 'react-router-dom';
import RecentEmployer from '../RecentEmployer';
import renderer from 'react-test-renderer';
import { initialState } from '../__mock__';
import configureMockStore from 'redux-mock-store';
import thunk from 'redux-thunk';
import { Provider } from 'react-redux';

let store: any;

const middleware = [thunk];

const mockStore = configureMockStore(middleware);

const mockSetPreviousStep = jest.fn();

describe('`RecentEmployer` component', () => {
  beforeEach(() => {
    store = mockStore(initialState);
  });

  afterEach(cleanup);

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDom.render(
      <Provider store={store}>
        <Router>
          <RecentEmployer setPreviousStep={mockSetPreviousStep} />
        </Router>
      </Provider>,
      div
    );
  });

  it('should create a snapshot', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Router>
            <RecentEmployer setPreviousStep={mockSetPreviousStep} />
          </Router>
        </Provider>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

});
