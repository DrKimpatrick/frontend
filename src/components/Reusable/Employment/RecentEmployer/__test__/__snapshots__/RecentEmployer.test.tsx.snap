// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`\`RecentEmployer\` component should create a snapshot 1`] = `
<div
  className="recent-employer-section m-auto text-gray-texts"
>
  <div
    className="containers"
  >
    <div
      className="flex relative h-auto my-8"
      style={
        Object {
          "flexDirection": "column",
        }
      }
    >
      <div
        style={
          Object {
            "display": "flex",
          }
        }
      >
        <h1
          className="font-bold text-base title"
          style={
            Object {
              "marginLeft": 0,
            }
          }
        >
          Who was your most recent employer?
        </h1>
      </div>
      <form
        autoComplete="off"
        onSubmit={[Function]}
      >
        <div
          className="text-gray-texts mt-2"
        >
          <input
            className="border outline-none bg-transparent rounded-sm w-full px-3 text-gray-texts input-height"
            name="companyName"
            onChange={[Function]}
            placeholder="Company Name *"
            type="text"
            value=""
          />
        </div>
        <div
          className="text-gray-texts mt-2"
        >
          <div
            className="customSelect"
            style={
              Object {
                "width": "100%",
              }
            }
          >
            <select
              name="employmentType"
              onChange={[Function]}
              value=""
            >
              <option
                value=""
              >
                Employment type *
              </option>
              <option
                value="Contract"
              >
                Contract
              </option>
              <option
                value="FullTime"
              >
                FullTime
              </option>
              <option
                value="PartTime"
              >
                PartTime
              </option>
            </select>
            <div
              className="customSelectArrow"
            >
              <div
                className="div"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                  />
                </svg>
              </div>
            </div>
          </div>
        </div>
        <div
          className="text-gray-texts mt-2"
        >
          <div
            className="customSelect"
            style={
              Object {
                "width": "100%",
              }
            }
          >
            <select
              name="supervisor.name"
              onChange={[Function]}
              value=""
            >
              <option
                value=""
              >
                Employment Reference *
              </option>
              <option
                value="Staffing"
              >
                Staffing
              </option>
              <option
                value="HR"
              >
                HR
              </option>
              <option
                value="Employee"
              >
                Employee
              </option>
              <option
                value="N/A"
              >
                N/A
              </option>
            </select>
            <div
              className="customSelectArrow"
            >
              <div
                className="div"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                  />
                </svg>
              </div>
            </div>
          </div>
        </div>
        <div
          className="text-gray-texts mt-2"
        >
          <input
            className="border outline-none bg-transparent rounded w-full px-3 text-gray-texts input-height"
            name="title"
            onChange={[Function]}
            placeholder="Your Title *"
            type="text"
            value=""
          />
        </div>
        <div
          className="flex justify-between text-gray-texts dateContainer"
        >
          <div
            className="item w-full h-full pt-1 pb-1 required"
          >
            <label
              className="required"
              htmlFor="start date"
            >
              Start date *
            </label>
            <input
              id="date"
              name="startDate"
              onChange={[Function]}
              type="date"
              value=""
            />
          </div>
          <div
            className="item w-full h-full pt-1 pb-1"
          >
            <label
              htmlFor="end date"
            >
              End date
            </label>
            <input
              id="date"
              name="endDate"
              onChange={[Function]}
              type="date"
              value=""
            />
          </div>
        </div>
        <div
          className="text-gray-texts flex"
          style={
            Object {
              "alignItems": "center",
            }
          }
        >
          <div
            style={
              Object {
                "flexGrow": 1,
                "width": "50%",
              }
            }
          />
          <div
            style={
              Object {
                "flexGrow": 1,
                "width": "50%",
              }
            }
          />
        </div>
        <div
          className="text-gray-texts flex"
          style={
            Object {
              "alignItems": "center",
            }
          }
        >
          <span
            aria-disabled={false}
            className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-1 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={null}
          >
            <span
              className="MuiIconButton-label"
            >
              <input
                aria-label="uncontrolled-checkbox"
                checked={false}
                className="PrivateSwitchBase-input-4"
                data-indeterminate={false}
                name="isCurrentPosition"
                onChange={[Function]}
                type="checkbox"
              />
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                />
              </svg>
            </span>
            <span
              className="MuiTouchRipple-root"
            />
          </span>
          <label
            htmlFor=""
          >
            I am currently working here
          </label>
        </div>
        <div
          className="text-gray-texts mt-2"
        >
          <div
            className="select css-2b097c-container"
            onKeyDown={[Function]}
          >
            <div
              className=" css-c8odyh-control"
              onMouseDown={[Function]}
              onTouchEnd={[Function]}
            >
              <div
                className=" css-g1d714-ValueContainer"
              >
                <div
                  className=" css-1wa3eu0-placeholder"
                >
                  Select skills used
                </div>
                <div
                  className="css-b8ldur-Input"
                >
                  <div
                    className=""
                    style={
                      Object {
                        "display": "inline-block",
                      }
                    }
                  >
                    <input
                      aria-autocomplete="list"
                      autoCapitalize="none"
                      autoComplete="off"
                      autoCorrect="off"
                      disabled={false}
                      id="react-select-3-input"
                      onBlur={[Function]}
                      onChange={[Function]}
                      onFocus={[Function]}
                      spellCheck="false"
                      style={
                        Object {
                          "background": 0,
                          "border": 0,
                          "boxSizing": "content-box",
                          "color": "inherit",
                          "fontSize": "inherit",
                          "label": "input",
                          "opacity": 1,
                          "outline": 0,
                          "padding": 0,
                          "width": "1px",
                        }
                      }
                      tabIndex="0"
                      type="text"
                      value=""
                    />
                    <div
                      style={
                        Object {
                          "height": 0,
                          "left": 0,
                          "overflow": "scroll",
                          "position": "absolute",
                          "top": 0,
                          "visibility": "hidden",
                          "whiteSpace": "pre",
                        }
                      }
                    >
                      
                    </div>
                  </div>
                </div>
              </div>
              <div
                className=" css-1hb7zxy-IndicatorsContainer"
              >
                <span
                  className=" css-1okebmr-indicatorSeparator"
                />
                <div
                  aria-hidden="true"
                  className=" css-tlfecz-indicatorContainer"
                  onMouseDown={[Function]}
                  onTouchEnd={[Function]}
                >
                  <svg
                    aria-hidden="true"
                    className="css-6q0nyr-Svg"
                    focusable="false"
                    height={20}
                    viewBox="0 0 20 20"
                    width={20}
                  >
                    <path
                      d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                    />
                  </svg>
                </div>
              </div>
            </div>
            <div>
              <input
                name="skillsUsed"
                type="hidden"
              />
            </div>
          </div>
        </div>
        <div
          className="text-gray-texts mt-2 divider"
          style={
            Object {
              "border": "none",
            }
          }
        >
          <div
            className="responsibilities addBorder"
          >
            <input
              className="outline-none bg-transparent rounded w-full px-3 text-gray-texts input-height"
              name="responsibility"
              onChange={[Function]}
              placeholder="Responsibilities"
              type="text"
              value=""
            />
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-text add"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                  />
                </svg>
              </span>
              <span
                className="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          className="text-gray-texts mt-2 divider"
          style={
            Object {
              "border": "none",
            }
          }
        >
          <div
            className="responsibilities addBorder"
          >
            <input
              className="outline-none bg-transparent rounded w-full px-3 text-gray-texts input-height"
              name="accomplishment"
              onChange={[Function]}
              placeholder="Accomplishments"
              type="text"
              value=""
            />
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-text add"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                  />
                </svg>
              </span>
              <span
                className="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          className="text-gray-texts mt-2"
        >
          <input
            className="border outline-none bg-transparent rounded w-full px-3 text-gray-texts input-height"
            name="favoriteProject"
            onChange={[Function]}
            placeholder="Favorite project you built or contributed to?"
            type="text"
            value=""
          />
        </div>
        <div
          className="flex justify-center mt-12"
        >
          <button
            className="next-btn text-white hover:bg-gray-800 font-semibold py-1 px-3 w-32 rounded-sm shadow flex justify-around"
            data-testid="next-button"
            disabled={false}
            type="submit"
          >
            <span>
              Next
            </span>
             
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M16.01 11H4v2h12.01v3L20 12l-3.99-4v3z"
              />
            </svg>
          </button>
        </div>
      </form>
    </div>
  </div>
</div>
`;
